app/models/spree/gateway/pay_pal_express.rb

```diff
@@ -71,18 +71,20 @@ def purchase(amount, express_checkout, gateway_options={})
         transaction_id = pp_response.do_express_checkout_payment_response_details.payment_info.first.transaction_id
         express_checkout.update_column(:transaction_id, transaction_id)
         # This is rather hackish, required for payment/processing handle_response code.
-        Class.new do
-          def success?; true; end
+        class << pp_response
           def authorization; nil; end
-        end.new
+          def to_s
+            do_express_checkout_payment_response_details
+          end
```

Comment:
Yes, we don't need it. I was trying something and forgot to save the file without it before commiting. I took it out.